services/galley/test/integration/API/Teams.hs

```diff
@@ -1053,6 +1063,88 @@ testDeleteBindingTeamSingleMember = do
   -- Let's clean the queue, just in case
   ensureQueueEmpty
 
+testDeleteTeamVerificationCodeSuccess :: TestM ()
+testDeleteTeamVerificationCodeSuccess = do
+  (owner, _) <- Util.createBindingTeam'
+  let Just email = U.userEmail owner
+  generateVerificationCode $ Public.SendVerificationCode Public.DeleteTeam email
+  pure ()
+
+testDeleteTeamVerificationCodeMissingCode :: TestM ()
+testDeleteTeamVerificationCodeMissingCode = do
+  g <- view tsGalley
+  (owner, tid) <- Util.createBindingTeam
+  setTeamSndFactorPasswordChallenge tid Public.TeamFeatureEnabled
+  delete
```

Comment:
as in the other PRs: always generate the code, even if you're not using it, or using it wrong.